common/djangoapps/student/models/user.py

```diff
@@ -1383,13 +1383,14 @@ def is_enabled(self, *key_fields):  # pylint: disable=arguments-differ
         share_settings = configuration_helpers.get_value('SOCIAL_SHARING_SETTINGS', settings.SOCIAL_SHARING_SETTINGS)
         return share_settings.get('CERTIFICATE_LINKEDIN', enabled)
 
-    def add_to_profile_url(self, course_name, cert_mode, cert_url, certificate=None):
+    def add_to_profile_url(self, course, cert_mode, cert_url, certificate=None):
+
         """
         Construct the URL for the "add to profile" button. This will autofill the form based on
         the params provided.
 
         Arguments:
-            course_name (str): The display name of the course.
+            course (str): Course Object.
```

Comment:
You are expecting the caller to pass a `CourseOverview` object, instead of a string, so the argument signature should be changed as well.
```suggestion
            course (CourseOverview): Course Object.
```

Reply:
@Cup0fCoffee Yes. I have made the fix in my latest change. Thank you for the support.

Reply:
The change hasn't been applied. Did you push the change to the branch?